### ISSUE TITLE
[Bug]:  Keeps asking for my osx credentials event though I clicked always allow

### ISSUE THREAD

**Issue by BradleyYeo** [denolehov/obsidian-git#742]
### Describe the bug

Whenever it is time to auto backup during git pull and push it asks for credentials

### Steps to reproduce

I set auto backup every 103 minutes

### Operating system

macOS

### Installation Method

None

### Plugin version

latest

**Comment by Vinzent03** [comment]
Does the same happen when you try to pull/push from the terminal multipletimes? The plugin itself doesn't handle authentication, but delegates this to the git cli and its configuration.

**Comment by adamwhitlock1** [comment]
same thing happening for me. 

If I pull from my terminal the repo updates just fine, but this plugin keeps triggering the permissions popup. I'm also on MacOS with the latest version of the plugin.

**Comment by BradleyYeo** [comment]
Nope, no problems from the terminal. Does the plugin disable git fetch by default/allow me to disable it? How should I configure my git cli to prevent this issue?Sent from my iPhoneOn 29 Jun 2024, at 7:26 PM, Vinzent ***@***.***> wrote:﻿
Does the same happen when you try to pull/push from the terminal multipletimes? The plugin itself doesn't handle authentication, but delegates this to the git cli and its configuration.

**Comment by Vinzent03** [comment]
I don't own a mac device and haven't heard of this issue before. Nothing changed in the plugin's source code, as well. So I currently don't know how to fix that, sorry.

**Comment by smudge** [comment]
Just as another data point, this started happening for me as well in the last week or so. I wonder if there was a change to macOS (or maybe the `git` homebrew package?) that could have resulted in a handful of us experiencing this.

**Comment by jeffscottward** [comment]
Im getting this issue too. I'm a recent user of Obsidian so I don't know if this was a recent problem.

**Comment by Vinzent03** [comment]
From Obsidian side I could only think of the [electron update](https://obsidian.md/changelog/2024-06-25-desktop-v1.6.5/), which is low level enough to may mess things up.

**Comment by Vinzent03** [comment]
That's very strange as git and the credentials access shouldn't be able to differentiate between manual and automatic. I can only think of Apple blocking the access, because you didn't type anything at that moment.
